radio/src/lua/api_general.cpp

```diff
@@ -2420,9 +2550,11 @@ const luaL_Reg opentxLib[] = {
   { "getGlobalTimer", luaGetGlobalTimer },
   { "getRotEncSpeed", luaGetRotEncSpeed },
   { "getValue", luaGetValue },
+  { "getSourceValue", luaGetSourceValue },
   { "getRAS", luaGetRAS },
   { "getTxGPS", luaGetTxGPS },
   { "getFieldInfo", luaGetFieldInfo },
+  { "getSourceInfo", luaGetFieldInfo },
```

Comment:
@jfrickmann Is the duplication intentional? Is that just for convenience of API?

Reply:
Yes, it is what Robert requested in pt. 2 here: https://github.com/EdgeTX/edgetx/issues/781#issuecomment-1193895564